src/iblt.h

```diff
@@ -135,11 +152,13 @@ class CIblt
 protected:
     void _insert(int plusOrMinus, uint64_t k, const std::vector<uint8_t> &v);
 
+    uint32_t salt;
```

Comment:
add a doc comment (//! or /** here) describing what salt does and why.  A subset of what you put in this PR description is what I'm looking for.

Reply:
Done.